app/routes/companyInterest/CompanyInterestListRoute.ts

```diff
@@ -6,16 +6,18 @@ import { fetchSemesters } from 'app/actions/CompanyActions';
 import {
   fetchAll,
   deleteCompanyInterest,
-  fetch,
+  fetch as fetchCIA,
```

Comment:
why

Reply:
I now see it's because you use node's `fetch`, but after moving that, rename this back.

Reply:
It has been removed now.